Sources/Accord.Imaging.WPF/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;

```

Comment:
Same here - start with a real copy of the Accord.Imaging project (copy the entire Accord.Imaging folder and rename it to Accord.Imaging.WPF), and then just adjust the fields that before contained the text "Accord.Imaging" to have "Accord.Imaging.WPF" instead.